### ISSUE TITLE
Feature Request: Multiple Tags per Talkgroup

### ISSUE THREAD

**Issue by dgwood12** [chuot/rdio-scanner#230]
I thought it would be nice to have the ability to add multiple tags per talkgroup. In my setup, I have several systems that are within my home county. Since they are separate systems, I have broken them out as such in rdio-scanner so my listeners can easily see the systems and which talkgroups belong to that system. I have the toggle by tags option enabled in my rdio-scanner instance and thought it would be nice to be able to add a second tag to all the talkgroups in one county with that county name. That way, listeners can easily turn on or off multiple systems that belong to that county with one button.

Thanks for all your hard work as always. Rdio-scanner is amazing!

**Comment by chuot** [comment]
Indeed, something like this is planned for Rdio Scanner v7 where a complete database refactoring is planned an which will allows such enhancement.
